### ISSUE TITLE
Scrobbling issues

### ISSUE THREAD

**Issue by deluan** [simojenki/bonob#9]
Hey @simojenki, I'm fainnaly adding [Last.fm scrobbling to Navidrome](https://github.com/navidrome/navidrome/pull/1204), and now that I took a deeper look on how bonob is scrobbling, I'm seeing some issues:

1) It is sending duplicated scrobbles. Every song I play in Sonos causes bonob to call `/rest/scrobble` twice. Maybe you meant to call one time as a "now playing" call? If that is the case, the "now playing" call should not pass `submission=true` as argument

2) It would be better to send the scrobble (`submission=true`) after the song is playing for a while, as documented here: https://www.last.fm/api/scrobbling#when-is-a-scrobble-a-scrobble I understand that this may not be feasible, as you don't have control on when the Sonos player is actually playing the song, so this is just a "nice to have"

3) It would be good to have an option to disable scrobbling altogether, as Sonos itself has support for it. 

4) Something we may not have control over: All scrobbles are sent to the same account in Last.fm. Do you think we could come up with way to identify the Navidrome user and pass different URLs (with different user/token) to the Sonos player?

Anyway, thanks again for this nice piece of software!

**Comment by simojenki** [comment]
Hi @deluan,

Did you get a chance to test the latest build?

In reply to your points; 
1. I'm not sure where the duplicates would come from, do you still see this after the latest changes?
2. I've changed bonob to send submission=false on play, and submission=true only after 30s of play.
3. You can disable scrobble using the env vars outlined in the README
4. From a bonob point of view we authenticate with the navidrome/subsonic api and send the scrobble.  The url for the scrobble already contain the user with the u query string param.
